build.gradle.kts

```diff
@@ -19,6 +18,7 @@ repositories {
     // - https://maven.enginehub.org/repo/
     // - https://maven.terraformersmc.com/releases/
     // - https://maven.minecraftforge.net/
+    maven(url = "https://jitpack.io")
```

Comment:
no, let me know what is needed from here and i'll publish it, rather than using this garbage.

Reply:
OptiFabric. Depends if we are going to support it or not, otherwise it wont be needed.

Reply:
we will do this